docs/source/tuning.rst

```diff
@@ -215,6 +215,9 @@ As a next step, we strongly decrease ``global_sampling_ratio`` and ``constraint_
 to compensate for the large number of constraints.
 We then tune for lower latency as explained above until the system reliably works in real time.
 
++If you run in ``pure_localization``, ``submaps.resultion`` **should be matching** with the ``.pbstream`` you are running on.
++Using different solutions are untested and might or might not work.
```

Comment:
suggestion:
Using different resolutions is currently untested and may not work as expected.